src/main/scala/com/lightbend/conductr/sbt/package.scala

```diff
@@ -92,7 +92,7 @@ package object sbt {
   object PlayBundleKeyDefaults {
     import com.lightbend.conductr.sbt.BundleImport.ByteConversions._
 
-    val nrOfCpus = 1.0
+    val nrOfCpus = 0.1
```

Comment:
This is the main change of the PR. The rest is just updating documentation and test data.